ui.c

```diff
@@ -68,6 +69,32 @@ void handleInput(int ch) {
   case KEY_UP:
     menu_driver(song_menu, REQ_UP_ITEM);
     break;
+  case '>':
```

Comment:
```suggestion
  case '>':
  case KEY_RIGHT:
```

If we're planning to traverse directory with arrow right this might not be useful but until then adding `KEY_RIGHT` seems... right